src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -141,7 +243,7 @@ public void WriteText(SourceText text, CancellationToken cancellationToken)
 
                         // the method we use to get text out of SourceText uses Unicode (2bytes per char). 
                         var size = Encoding.Unicode.GetMaxByteCount(text.Length);
-                        _memoryMappedInfo = new MemoryMappedInfo(size);
+                        _memoryMappedInfo = _service.CreateTemporaryStorage(size);
```

Comment:
I think my thought for the name would be `MemoryMappedBlock` or `MemoryMappedAllocation` or something like that. I've been bugged by the name too.